### PR TITLE
[STARTAI-46] Mobile-Pricing | Create classifier

### PR DESCRIPTION
**Pull request by nasstis**

### CODE REVIEW

File: classifier/logistic_regression.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        self.n_iters = n_iters
+        self.classifiers = {}
+
+    def fit(self, X, y):
+        unique_classes = np.unique(y)
+        for cls in unique_classes:
+            binary_y = np.where(y == cls, 1, 0)
+            self.classifiers[cls] = self.train_classifier(X, binary_y)
+
+    def train_classifier(self, X, y):
```
Comment (JJerome-NM) [inline classifier/logistic_regression.py]: Make this field private

`__train_classifier(self, X, y):`

File: classifier/logistic_regression.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+def sigmoid(x):
+    return 1 / (1 + np.exp(-x))
+
+class LogisticRegression:
+    
+    def __init__(self, learning_rate=0.001, n_iters=1000):
+        self.learning_rate = learning_rate
```
Comment (JJerome-NM) [inline classifier/logistic_regression.py]: I think these fields should also be made private

File: classifier/logistic_regression.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        self.n_iters = n_iters
+        self.classifiers = {}
+
+    def fit(self, X, y):
+        unique_classes = np.unique(y)
+        for cls in unique_classes:
```
Comment (JJerome-NM) [inline classifier/logistic_regression.py]: `for cls in np.unique(y):`

File: classifier/logistic_regression.py (hunk 1 above)
Comment (ArsenOOO7) [reply classifier/logistic_regression.py:19]: to take

File: classifier/logistic_regression.py (hunk 3 above)
Comment (ArsenOOO7) [reply classifier/logistic_regression.py:15]: to take

**Review by ArsenOOO7 — APPROVED** [review]

**Review by JJerome-NM — APPROVED** [review]

**Pull request merged**